### PR TITLE
Pods 7670

### PR DESCRIPTION
**Pull request by jamdade-sharad**

### CODE REVIEW

File: app/controllers/common/TotalPensionAmountsController.scala [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+class TotalPensionAmountsController @Inject()(val controllerComponents: MessagesControllerComponents,
+                                              identify: IdentifierAction,
+                                              getData: DataRetrievalAction,
+                                              userAnswersCacheConnector: UserAnswersCacheConnector,
+                                              formProvider: TotalPensionAmountsFormProvider,
+                                              view: TotalPensionAmountsView
+                                     )(implicit ec: ExecutionContext) extends FrontendBaseController with I18nSupport {
+
+  private val form = formProvider()
+//  private val eventType = EventType.Event23
```
Comment (grantrobson) [inline app/controllers/common/TotalPensionAmountsController.scala:44]: Remove this

File: app/viewmodels/checkAnswers/ChooseTaxYearSummary.scala [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+object ChooseTaxYearSummary {
+
+  def row(answers: UserAnswers, waypoints: Waypoints, sourcePage: CheckAnswersPage, eventType: EventType)
+         (implicit messages: Messages): Option[SummaryListRow] =
+    answers.get(common.ChooseTaxYearPage(eventType)).map {
+      answer =>
+
+        val value = ValueViewModel(
+          HtmlContent(
+            HtmlFormat.escape(answer.toString + " to " + (answer.toString.toInt + 1).toString)
```
Comment (grantrobson) [inline app/viewmodels/checkAnswers/ChooseTaxYearSummary.scala:38]: This content should really be in the messages file

File: app/controllers/common/TotalPensionAmountsController.scala (hunk 1 above)
Comment (jamdade-sharad) [reply app/controllers/common/TotalPensionAmountsController.scala]: removed commented code

File: app/viewmodels/checkAnswers/ChooseTaxYearSummary.scala (hunk 2 above)
Comment (jamdade-sharad) [reply app/viewmodels/checkAnswers/ChooseTaxYearSummary.scala]: fixed

**Review by grantrobson — APPROVED** [review]

File: app/controllers/event23/Event23CheckYourAnswersController.scala [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  def onPageLoad(): Action[AnyContent] =
+    (identify andThen getData(Event23) andThen requireData) { implicit request =>
+      val thisPage = Event23CheckYourAnswersPage
+      val waypoints = EmptyWaypoints
+      val continueUrl = controllers.event23.routes.Event23CheckYourAnswersController.onClick.url
+      Ok(view(SummaryListViewModel(rows = buildEvent23CYARows(waypoints, thisPage)), continueUrl))
+    }
+
+  /**
+   *TODO: replace controllers.event23.routes.Event23CheckYourAnswersController with actual event23.SummaryPageController
```
Comment (cyeniceri) [inline app/controllers/event23/Event23CheckYourAnswersController.scala:55]: what do you mean by this ? Why do we have on click endpoint ?

**Review by cyeniceri — CHANGES_REQUESTED** [review]

File: app/controllers/event23/Event23CheckYourAnswersController.scala (hunk 3 above)
Comment (jamdade-sharad) [reply app/controllers/event23/Event23CheckYourAnswersController.scala]: As we have common view file and separate controllers for event1, event23 etc. to sync with view added onClick endpoint.
and Summary page for event23 after CYA page not yet developed, so added TODO "to replace dummy code with SummaryController" once ready for event23

**Review by cyeniceri — APPROVED** [review]

**Pull request merged**